pom.xml

```diff
@@ -748,6 +748,34 @@
                     </plugin>
                 </plugins>
             </build>
+            <modules>
+                <module>bvm/ballerina-core</module>
+                <module>bvm/ballerina-logging</module>
+                <module>cli/ballerina-launcher</module>
+                <module>cli/ballerina-packerina</module>
+                <module>compiler/ballerina-lang</module>
```

Comment:
The compiler module should be the first module here.  Even though Maven reorders this, it is clean if we order it manually.

Reply:
I did it in ascending order, :) will change and update

Reply:
I see :) 

It looks cleaner if the module order in this pom.xml is same as the actual dependency order.

Reply:
changed